### ISSUE TITLE
Cannot install or upgrade

### ISSUE THREAD

**Issue by kevwil** [dylang/npm-check#127]
Not sure if install has a bad symlink or what. Previous version worked fine.

https://gist.github.com/kevwil/1b5b35282c7845a6f5c807fed7261bdd

**Comment by lijunle** [comment]
@kevwil have you run with `sudo`?

**Comment by kevwil** [comment]
No, my /usr/local is owned by my user.

I just tried installing 5.1.1 and it failed as well, but 5.0.3 installed just fine.

**Comment by chrisalcantara** [comment]
Same issue.

**Comment by dylang** [comment]
@chrisalcantara I just released 5.2.0, can you give that one a try? 

Also try `npm cache clean`.

**Comment by chrisalcantara** [comment]
Unfortunately, that didn't work. This is the error message I'm getting when I `sudo install`. 

```
/usr/local/lib
└── (empty)

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.8.0/bin/node" "/usr/local/bin/npm" "install" "-g" "npm-check"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! path /usr/local/lib/node_modules/npm-check/bin/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/npm-check/bin/cli.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/npm-check/bin/cli.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/npm-debug.log
npm ERR! code 1
```

**Comment by jamg44** [comment]
Same here

**Comment by nsanthoshkumar** [comment]
Same here. I am not able to install it

**Comment by sambernard** [comment]
Just ran into this as well. Both on my Mac, and in Docker.

**Comment by jameswomack** [comment]
Regardless of the version of Node or NPM I am using, I experience the issue of not being able to install or upgrade

```zsh
> $ npm i npm-check -g                                                                                                              ⬡ 5.10.1 [±cleanup/deps-general ●●●]
/Users/myuser/.nvm/versions/node/v5.10.1/lib
└── (empty)

npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/myuser/.nvm/versions/node/v5.10.1/bin/node" "/Users/myuser/.nvm/versions/node/v5.10.1/bin/npm" "i" "npm-check" "-g"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! path /Users/myuser/.nvm/versions/node/v5.10.1/lib/node_modules/npm-check/bin/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/myuser/.nvm/versions/node/v5.10.1/lib/node_modules/npm-check/bin/cli.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/myuser/.nvm/versions/node/v5.10.1/lib/node_modules/npm-check/bin/cli.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/myuser/Projects/github/jameswomack/react-localize/npm-debug.log
npm ERR! code 1
```

**Comment by SpencerCarstens** [comment]
Exact same issue. :+1:

**Comment by abernix** [comment]
`npm-check` has been working for me for a while.  The `5.0` jump broke for me.  `4.1.4` still installs and works fine.

`npm ERR! Please include the following file with any support request:`
Here is my [npm-debug.log](https://gist.github.com/abernix/9c89e20edb09798c25cfac579c2fe4a5)

**Comment by robmcguinness** [comment]
```bash
npm install npm-check@5.0.1 -g
```

worked for me

**Comment by dylang** [comment]
Thanks everyone, fixed in 5.2.1!

```
npm -g i npm-check 
```

to get the latest. Let me know if you are still having problems.

**Comment by chrisalcantara** [comment]
Works now. Thanks so much!
